components/transactions/TransactionsDownloadCSV.js

```diff
@@ -44,11 +44,14 @@ const transformResultInCSV = json => {
     `Net Amount (${collectiveCurrency})`,
     'Subscription Interval',
     'Order Date',
+    'Tags',
   ].join(',');
 
   const lines = json.map(i => {
     const profile = `http://opencollective.com/${i.fromCollective.slug}`;
     const subscriptionInterval = i.subscription ? i.subscription.interval : 'one time';
+    const expenseTags = i.expense && i.expense.tags ? i.expense.tags.join(', ') : '';
```

Comment:
Would do a minor simplification, 

```suggestion
    const expenseTags = i.expense?.tags ? i.expense.tags.join(', ') : '';
```